requirements.txt

```diff
@@ -1,2 +1 @@
 .
--e git+https://github.com/CartoDB/carto-python.git#egg=carto
```

Comment:
We still need this. I am going to create a PR with that

Reply:
This PR does not need to point to carto-python master.

Reply:
But you are merging into develop and the code there needs master